easyrsa3/easyrsa

```diff
@@ -674,6 +680,79 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
             fi
         fi
 	fi
+
+	# Generate the CA with AES256
+
+	# => BEGIN SSL lib version
+	case "$osslv_major" in
+
+	# BEGIN SSL V3
+	3)
+		# OpenSSL v3: '-nodes' is deprecate, use '-noenc'
+		unset -v no_password; [ -z "$nopass" ] || no_password='-noenc'
+
+		# Generate CA
+		case "$EASYRSA_ALGO" in
+		rsa)
+			# OpenSSL v3: 'genrsa' is deprecate, use 'req'
+				easyrsa_openssl req -utf8 "${no_password}" \
+					-newkey "$EASYRSA_ALGO":"$EASYRSA_KEY_SIZE" \
+					-keyout "$out_key_tmp" -out "$out_file_tmp" \
+					${opts} ${crypto_opts} \
+					${EASYRSA_PASSOUT:+-passout "$EASYRSA_PASSOUT"} || \
+						die "Failed to build the CA" ;;
+		ec)
+			# EC params
+			param_file="$(easyrsa_mktemp)"
+			easyrsa_openssl ecparam -name "$EASYRSA_CURVE" > "$param_file"
+
+				easyrsa_openssl req -utf8 "${no_password}" \
```

Comment:
indentation is now all over the place. But to be fair, that is true of the old code as well...

Reply:
In order to keep this patch lean, I am not indenting OpenSSL version 1 commands.  However, I can do that in this patch if preferred.